src/index.js

```diff
@@ -452,6 +452,11 @@ module.exports = {
 					urlPath = urlPath.replace(this._isscRe, "$");
 					let action = urlPath;
 
+					// #331 Always merge parameters for internal services
+					if (urlPath.indexOf("$") !== -1) {
```

Comment:
I don't know if `$` is always at the start of internal actions but I would have use `startsWith` instead if it's the case (I think it is but I'm not sure).
If it's not I would have used `action.includes("$")` as it's more readable.

```suggestion
					if (action.startsWith("$")) {
```